lib/DevTools.ts

```diff
@@ -58,7 +58,7 @@ class DevTools {
      * @param payload - data written to the storage
      * @param stateChanges - partial state that got updated after the changes
      */
-    registerAction(type: string, payload: unknown, stateChanges: Record<string, unknown> = {}) {
+    registerAction(type: string, payload: unknown, stateChanges: Record<string, unknown> | null = {}) {
         try {
             if (!this.remoteDev) {
                 return;
```

Comment:
NAB: if `payload` can be `null` (or even an empty object `{}`), the state will remain the same right? shouldn't we just return instead of creating a copy and setting it as a new state?

```
{...{a: 1}, ...null}
{a: 1}
```

NAB because I have no idea of what DevTools.ts is and maybe this matters very little